README.adoc

```diff
@@ -256,7 +254,7 @@ Navigate your browser to the http://localhost:9080/ServletSample/servlet[^] URL
 == Testing the web application
 
 One of the benefits of building an application with Maven is that Maven can be configured 
-to run a set of tests. You can write tests for the individual units of code outside of a 
+to run a set of predefined tests. You can write tests for the individual units of code outside of a 
```

Comment:
I'm not sure that this extra word adds anything to the meaning. Can you go back to the original?